src/overlay/components/visual/Clock.scss

```diff
@@ -1,6 +1,8 @@
 .clock {
     font-variant-numeric: tabular-nums;
 
+    margin-right: 2em;
```

Comment:
Is the clock always going to have this margin? The clock styles should be the bare minimum to get the clock looking right. Any outside margins should probably be applied by the container. That way if we move the clock to a different container, the margin doesn't follow it.